internal/util/grpcclient/client.go

```diff
@@ -328,12 +341,16 @@ func (c *ClientBase[T]) call(ctx context.Context, caller func(client T) (any, er
 		}
 		ret, callErr = caller(client)
 		if callErr != nil {
-			if funcutil.IsGrpcErr(callErr) ||
+			if funcutil.IsGrpcErr(callErr, codes.Unavailable, codes.Aborted, codes.FailedPrecondition) ||
 				IsCrossClusterRoutingErr(callErr) || IsServerIDMismatchErr(callErr) {
 				log.Warn("start to reset connection because of specific reasons", zap.Error(callErr))
 				resetClientFunc()
 				return callErr
 			}
+			if funcutil.IsGrpcErr(callErr) {
+				log.Warn("call received grpc error, no reset, retrying...", zap.Error(callErr))
+				return callErr
+			}
 			if !funcutil.CheckCtxValid(ctx) {
 				if c.sess != nil {
 					sessions, _, getSessionErr := c.sess.GetSessions(c.GetRole())
```

Comment:
this is might also be too frequent. 

We need to watch session in each querynode. and decide whether reset connection by if session exist